nginx/metadata.csv

```diff
@@ -3,103 +3,177 @@ nginx.net.writing,gauge,,connection,,The number of connections waiting on upstre
 nginx.net.waiting,gauge,,connection,,The number of keep-alive connections waiting for work.,0,nginx,conns waiting
 nginx.net.reading,gauge,,connection,,The number of connections reading client requets.,0,nginx,conns reading
 nginx.net.connections,gauge,,connection,,The total number of active connections.,0,nginx,conns
+nginx.net.connections_count,count,,connection,,The total number of active connections (shown as count).,0,nginx,conns
```

Comment:
total number of **active** connections: this can go up and down so it should not be a count